app/views/providers/check_benefits/index.html.erb

```diff
@@ -1,16 +1,17 @@
 <% result_namespace = @legal_aid_application.applicant_receives_benefit? ? ".positive_result" : ".negative_result" %>
 
-<div class="interruption-panel">
-  <%= page_template(page_title: t("#{result_namespace}.tab_title"), column_width: :full, template: :basic) do %>
-    <h1 class="govuk-heading-l">
-      <%= t("#{result_namespace}.title") %>
-    </h1>
-    <%= next_action_buttons_with_form(
-          url: providers_legal_aid_application_check_benefit_path,
-          method: :patch,
-          show_draft: true,
-          inverse_continue: true,
-          continue_button_text: t("generic.continue"),
-        ) %>
-  <% end %>
+<div class="moj-interruption-card">
+  <div class="moj-interruption-card__content">
+    <%= page_template(page_title: t("#{result_namespace}.tab_title"), template: :basic) do %>
+      <h1 class="govuk-heading-l moj-interruption-card__heading"><%= t("#{result_namespace}.title") %></h1>
+
+      <%= next_action_buttons_with_form(
+            url: providers_legal_aid_application_check_benefit_path,
+            method: :patch,
+            show_draft: true,
+            inverse_continue: true,
+            continue_button_text: t("generic.continue"),
+          ) %>
+    <% end %>
```

Comment:
This seem to be missing the parent element `<div class="govuk-button-group moj-interruption-card__actions">`